src/main/java/io/quarkus/bot/release/ReleaseInformation.java

```diff
@@ -175,9 +188,24 @@ public boolean isLtsMaintenanceReleaseWithRegularReleaseCadence() {
         return Branches.isLtsBranchWithRegularReleaseCadence(branch) && isFinal() && !isFirstFinal();
     }
 
+    @JsonIgnore
+    public void checkConsistency() {
```

Comment:
Any reason to not do this straight in the constructor? This is an immutable class so all necessary information is already known there. This way we may forget to call the method when it would be appropriate

Reply:
This is not completely immutable (mostly for historical reasons), we should probably have 2 different objects and make them immutable (basically, until the `Prerequisites` step, the information are partial and they are augmented during the `Prerequisites` step). But I never took the time to do it.

That being said, we are only using properties that are defined always so makes sense to move it.

Done!

Reply:
I created https://github.com/quarkusio/conversational-release-action/issues/187 to track this.